spec/rubocop/cop/performance/count_spec.rb

```diff
@@ -86,6 +86,20 @@
       expect(cop.highlights).to eq(["#{selector}(&:something).count"])
     end
 
+    it "registers an offense for #{selector}(&:something).count " \
```

Comment:
I don't see anything in the changed code that take into account the superclass, so this description isn't precise IMO.